paper-badge.html

```diff
@@ -183,6 +187,7 @@
         var targetRect = this._target.getBoundingClientRect();
         var thisRect = this.getBoundingClientRect();
 
+        this.classList.add('visible');
```

Comment:
No, `classList.add` will only add the class once.

The implementation on https://developer.mozilla.org/en/docs/Web/API/Element/classList will only push the class if it did not exists before
```javascript
classListProto.add = function () {
  var
      tokens = arguments
    , i = 0
    , l = tokens.length
    , token
    , updated = false
  ;
  do {
    token = tokens[i] + "";
    if (checkTokenAndGetIndex(this, token) === -1) {
      this.push(token);
      updated = true;
    }
  }
  while (++i < l);

  if (updated) {
    this._updateClassName();
  }
};
```